_episodes/11-using-arrays-transformations.md

```diff
@@ -21,31 +21,28 @@ Arrays can be sorted, de-duplicated, and manipulated in other ways in GREL expre
 
 "Monday,Tuesday,Wednesday,Thursday,Friday,Saturday,Sunday"
 
-This can be transformed into an array using the ```split``` function
+This can be transformed into an array using the ```split``` function with pipe
 ```
-value.split(",")
+value.split("|")
```

Comment:
The character in the split command (here the pipe) has to be the one that is used as the separator for values in the original textual/string